### DISCUSSION TITLE
Issue with vulkan-1.dll. for imagewriter/ffmpeg video recording

### DISCUSSION THREAD

**Discussion by Lindsey-Christinger** [bonsai-rx/bonsai#1854] [Q&A]
Hello, I am trying to record an mp4 video along with audio and timestamps. The workflow I am using works on one computer in the lab but the desktop I need it on successfully saves audio and timestamp but not video. They are both windows 7. I am using image writer and ffmpeg for the video recording and inconsistently get the error: the procedure entry point vkBindImageMemory2 could not be located in the dynamic link library vulkan-1.dll. Any help would be appreciated, thank you!

**Comment by bruno-f-cruz** [comment]
Can you share the workflow? In general, windows 7 is not really guaranteed to work anymore, so I would give win10 a try.

**Reply by Lindsey-Christinger** [reply]
Yes! Here is the workflow:
![image](https://github.com/bonsai-rx/bonsai/assets/112510219/fd501b77-21c2-4fbd-89b2-00df0e3e91b6)
[For_github.zip](https://github.com/user-attachments/files/15809125/For_github.zip)
I'm hoping to make it work on the desktops we have if possible. Thank you!

**Reply by bruno-f-cruz** [reply]
Given the nature of the dll, it is either being used during visualization of images or by ffmpeg. Can you disable the ffmpeg node and try to use the workflow? Do you still get the error?

**Reply by Lindsey-Christinger** [reply]
No, when I disable ffmpeg I don't get the error.

**Reply by bruno-f-cruz** [reply]
Then I am afraid that the error might be being thrown by ffmpeg itself. Have you tried to make sure:
1. Ffmpeg build is the same across computers
2. Same gpu/ driver version
3. Same windows update version
